travelnet.lua

```diff
@@ -1,40 +1,23 @@
 
--- "default" travelnet box in yellow
-travelnet.register_travelnet_box({
-	nodename = "travelnet:travelnet",
-	recipe = travelnet.travelnet_recipe,
-	color = "#e0bb2d",
-	dye = "dye:yellow"
-})
+local default_travelnets = {
+	-- "default" travelnet box in yellow
+	{ nodename="travelnet:travelnet", color="#e0bb2d", dye="dye:yellow", recipe=travelnet.travelnet_recipe },
+	{ nodename="travelnet:travelnet_red", color="#ce1a1a", dye="dye:red" },
+	{ nodename="travelnet:travelnet_orange", color="#ff8800", dye="dye:orange" },
+	{ nodename="travelnet:travelnet_blue", color="#0051c5", dye="dye:blue" },
+	{ nodename="travelnet:travelnet_cyan", color="#00eeee", dye="dye:cyan" },
+	{ nodename="travelnet:travelnet_green", color="#53c41c", dye="dye:green" },
+	{ nodename="travelnet:travelnet_dark_green", color="#33a40c", dye="dye:dark_green" },
+	{ nodename="travelnet:travelnet_violet", color="#ee82ee", dye="dye:violet" },
+	{ nodename="travelnet:travelnet_pink", color="#ffc0cb", dye="dye:pink" },
+	{ nodename="travelnet:travelnet_magenta", color="#ff0090", dye="dye:magenta" },
+	{ nodename="travelnet:travelnet_brown", color="#964b00", dye="dye:brown" },
+	{ nodename="travelnet:travelnet_grey", color="#4f4f4f", dye="dye:grey" },
+	{ nodename="travelnet:travelnet_dark_grey", color="#2f2f2f", dye="dye:dark_grey" },
```

Comment:
```suggestion
	{ nodename="travelnet:travelnet_orange", color="#e2621b", dye="dye:orange" },
	{ nodename="travelnet:travelnet_blue", color="#0051c5", dye="dye:blue" },
	{ nodename="travelnet:travelnet_cyan", color="#00a6ae", dye="dye:cyan" },
	{ nodename="travelnet:travelnet_green", color="#53c41c", dye="dye:green" },
	{ nodename="travelnet:travelnet_dark_green", color="#2c7f00", dye="dye:dark_green" },
	{ nodename="travelnet:travelnet_violet", color="#660bb3", dye="dye:violet" },
	{ nodename="travelnet:travelnet_pink", color="#ff9494", dye="dye:pink" },
	{ nodename="travelnet:travelnet_magenta", color="#d10377", dye="dye:magenta" },
	{ nodename="travelnet:travelnet_brown", color="#572c00", dye="dye:brown" },
	{ nodename="travelnet:travelnet_grey", color="#a2a2a2", dye="dye:grey" },
	{ nodename="travelnet:travelnet_dark_grey", color="#3d3d3d", dye="dye:dark_grey" },
```
![screenshot_20210713_2326](https://user-images.githubusercontent.com/48543043/125459987-53f24ead-0f0e-448e-a71d-f84c147758e7.png)